src/main/resources/application.yaml

```diff
@@ -0,0 +1,4 @@
+spring:
+  sql:
+    init:
+      mode: always
```

Comment:
Are you sure that is set right **init mode** ? 
Take a look onto Spring Database Initialization